src/de/jost_net/JVerein/gui/view/MailVorlageDetailView.java

```diff
@@ -54,4 +66,49 @@ public void handleAction(Object context)
     }, null, true, "document-save.png");
     buttons.paint(this.getParent());
   }
+
+  @Override
+  public void handleEvent(Event event)
+  {
+     if (event.keyCode == org.eclipse.swt.SWT.F6)
+    {
+      new OpenInsertVariableDialogAction().handleAction(null);
+    }
+  }
+
+  private static final class OpenInsertVariableDialogAction implements Action
```

Comment:
Ich fände es besser, wenn wir es einheitlich so machen, dass in der View wirklich nur das Aussehen ist und die Action in einer extra Klasse.

Reply:
In den Lesefeldern ist das genauso implementiert.

Reply:
Ja, das stimmt. Auch da werde ich es nochmal ändern. Bei den lesefeldern ist einiges nicht nach unserem normalen Schema...

Reply:
Dann kann ich es auch refactoren und zwar so, dass in beiden Fällen die selbe Action verwendet wird